vnext/ReactUWP/Views/SliderViewManager.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <Views/ShadowNodeBase.h>
+#include "SliderViewManager.h"
+
+#include <Utils/ValueUtils.h>
+
+#include <IReactInstance.h>
+
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+namespace winrt {
+using ToggleButton = Windows::UI::Xaml::Controls::Primitives::ToggleButton;
+}
+
+namespace react {
+namespace uwp {
+
+class SliderShadowNode : public ShadowNodeBase {
+  using Super = ShadowNodeBase;
+
+ public:
+  SliderShadowNode() = default;
+  void createView() override;
+  void updateProperties(const folly::dynamic &&props) override;
+};
+
+void SliderShadowNode::createView() {
+  Super::createView();
+
+  auto slider = GetView().as<winrt::Slider>();
+  auto wkinstance = GetViewManager()->GetReactInstance();
+}
+
+void SliderShadowNode::updateProperties(const folly::dynamic &&props) {
+  m_updating = true;
+  Super::updateProperties(std::move(props));
+  m_updating = false;
+}
+
+SliderViewManager::SliderViewManager(
+    const std::shared_ptr<IReactInstance> &reactInstance)
+    : Super(reactInstance) {}
+
+const char *SliderViewManager::GetName() const {
+  return "RCTSlider";
+}
+
+folly::dynamic SliderViewManager::GetNativeProps() const {
+  auto props = Super::GetNativeProps();
+
+  props.update(
+      folly::dynamic::object("value", "integer")("disabled", "boolean"));
+
+  return props;
+}
+
+facebook::react::ShadowNode *SliderViewManager::createShadow() const {
+  return new SliderShadowNode();
+}
+
+XamlView SliderViewManager::CreateViewCore(int64_t tag) {
+  auto slider = winrt::Slider();
+  slider.MinHeight(100);
```

Comment:
where does this number come from?  And doesn't slider already have a height in the default template?